### PR TITLE
Docs/v1.1 feedback

### LINKED ISSUE #80: Feedback for v1.1
- [x] Remove [W.I.P] tag
- [x] Include responsibility 
(UI, Git,  Code consistency, storage, testing, integration, model, deliverables and deadline)
- [x] Elaborate on User Guide introduction

Reference from https://github.com/nus-cs2103-AY1819S1/addressbook-level4/pull/21
- [x] Developer guide [Delete module > Use case > Extension](https://github.com/CS2103-AY1819S1-W17-2/main/blob/master/docs/DeveloperGuide.adoc#use-case-delete-module)
- [x] Developer guide [Product Survey](https://github.com/CS2103-AY1819S1-W17-2/main/blob/master/docs/DeveloperGuide.adoc#appendix-g-product-survey
)
- [x] Developer guide [Non functional requirements](https://github.com/CS2103-AY1819S1-W17-2/main/blob/master/docs/DeveloperGuide.adoc#appendix-e-non-functional-requirements)
- [x] User guide [addon description (Description and format)](https://github.com/CS2103-AY1819S1-W17-2/main/blob/master/docs/UserGuide.adoc#32-adding-a-module-addon-wip)
- [x] User guide [remove description (Description and format)](https://github.com/CS2103-AY1819S1-W17-2/main/blob/master/docs/UserGuide.adoc#33-deleting-a-module-remove-wip)
- [x] User guide [load command(format)](https://github.com/CS2103-AY1819S1-W17-2/main/blob/master/docs/UserGuide.adoc#36-loading-module-configurations-load-coming-in-v20
)
- [x] User guide [save command(description)](
https://github.com/CS2103-AY1819S1-W17-2/main/blob/master/docs/UserGuide.adoc#35-saving-module-configurations-save)

### PR DESCRIPTION
**Pull request by etjk**
Resolves #80

### CODE REVIEW

File: docs/AboutUs.adoc [hunk 1]
```diff
@@ -24,7 +24,7 @@ image::etjk.png[width="150", align="left"]
 {empty}[http://github.com/etjk[github]] [<<edricteo#, portfolio>>]
 
 Role: Developer +
-Responsibility: In charge of project documentation and ensures that all documentations are in place and updated.
+Responsibilities: In charge of Storage, Deliverables and deadlines
```
Comment (CrimsonJacket) [inline docs/AboutUs.adoc]: Should `deadlines` start with a capital?

File: docs/AboutUs.adoc [hunk 2]
```diff
@@ -51,6 +51,6 @@ image::rtjl.png[width="150", align="left"]
 {empty}[http://github.com/RTJL[github]] [<<reubentan#, portfolio>>]
 
 Role: Developer +
-Responsibility: In charge of project code quality and ensures that the codebase is seamless.
+Responsibilities: In charge of Integration, Code consistency
```
Comment (CrimsonJacket) [inline docs/AboutUs.adoc]: Also for `consistency` should it start with a capital?

**Comment by RTJL** [comment]
Looks good.

**Comment by hzf1174** [comment]
Looks good.

File: docs/UserGuide.adoc [hunk 3]
```diff
@@ -18,7 +18,12 @@ By: `Team W17-2`      Since: `Sep 2018`      Licence: `MIT`
 
 == Introduction
 
-ModUNI is an application for NUS students who want to quickly generate a plan for the modules to take during their entire university life. Interested? Jump to the <<Quick Start>> to get started. Enjoy!
+ModUNI is an application for NUS students who want to quickly generate a plan for the modules to take during their entire university life. +
```
Comment (RTJL) [inline docs/UserGuide.adoc]: Are we using "modsUni" or "ModUNI"? AboutUs.adoc uses modsUni. Might want to standardise that. @CS2103-AY1819S1-W17-2/developers

Comment (etjk) [reply docs/UserGuide.adoc]: I suggest using modsUni.

File: docs/UserGuide.adoc [hunk 4]
```diff
@@ -76,7 +81,7 @@ Examples:
 * `remove Cs2103t`
 * `remove CS2103T CS2101 CS1010` `[coming in v2.0]`
 
-=== Searching for a module: `search` `[W.I.P]`
+=== Searching for a module: `search` `[coming in v2.0]`
```
Comment (CrimsonJacket) [inline docs/UserGuide.adoc]: Nice to see some standardization.

File: docs/DeveloperGuide.adoc [hunk 5]
```diff
@@ -956,30 +949,30 @@ Use case ends.
...
+.  Should be able to generate a user’s schedule under less than 10 seconds.
+.  Sensitive information (e.g. passwords) should not be saved in plaintext.
+.  A user should be able to remove any personal identifiable information **(PII)** from the application.
+.  An administrator should not be able to log in on behalf of a user.
+
 
 [appendix]
 == Glossary
 
+[[cli]] CLI::
```
Comment (CrimsonJacket) [inline docs/DeveloperGuide.adoc]: Nice inclusion of certain terminologies in the Glossary.

**Pull request merged**